ledger/ledger-api-test-tool/src/main/scala/com/daml/ledger/api/testtool/suites/RaceConditionIT.scala

```diff
@@ -266,21 +245,45 @@ final class RaceConditionIT extends LedgerTestSuite {
         .getOrElse(fail("No create-non-transient transaction found"))
 
       transactions
-        .filter(isSuccessfulContractLookup(success = false))
+        .filter(isContractLookup(success = false))
         .foreach(assertTransactionOrder(_, createNonTransientTransaction))
     }
   }
 
+  private def executeRandomly[T](attempts: Int, once: => Future[T], repeated: => Future[T])(implicit
```

Comment:
Perhaps `executeRepeatedlyWithRandomDelay`?

I know it's a mouthful but "randomly" doesn't suggest a delay to me. If I couldn't see the implementation I would have assumed it was shuffling something.

Reply:
I second that opinion

Reply:
That's a good point! Fixed.